libs/cardano-ledger-core/src/Cardano/Ledger/Keys/Bootstrap.hs

```diff
@@ -208,7 +163,7 @@ makeBootstrapWitness ::
   Byron.Attributes Byron.AddrAttributes ->
   BootstrapWitness
 makeBootstrapWitness txBodyHash byronSigningKey addrAttributes =
-  BootstrapWitness vk signature cc (serialize' byronProtVer addrAttributes)
+  BootstrapWitness vk signature cc (serialize' addrAttributes)
```

Comment:
But how does this work without the version? It seems to be the same function as before??

Reply:
It's being imported from `Plain` now. (see the module where the function is being imported from has changed